runtime/src/main/java/org/corfudb/runtime/object/VersionLockedObject.java

```diff
@@ -668,4 +687,17 @@ public void applyUpdateToStreamUnsafe(SMREntry entry, long globalAddress) {
         seek(globalAddress + 1);
     }
 
+    /**
+     * This class includes the metrics registry and the timer names used within VersionLockedObject
+     * methods
+     */
+    private static class VloMetricsParams {
+        private static final MetricRegistry metrics = CorfuRuntime.getDefaultMetrics();
```

Comment:
Can we rename  `VloMetricsParams` to `VloMetrics`? For me `Params` means that the class has only parameters and doesn't contain anything else. in this case we also have `MetricRegistry` in the class, so wether to remove MetricRegistry or rename the class

Reply:
Metrics has metrics. changed to helper.